lib/main.js

```diff
@@ -19,6 +19,8 @@ const endpoints = Object.freeze({
     BUNDLE: 'bundle',
 });
 
+const assetTypes = Object.freeze(['js', 'css']);
```

Comment:
Using freeze here seems overkill, who will ever have access to edit this?

Reply:
Was just kind of a best practice we used at my last job. Happy to remove in favour of pragmatism